arcgis-ios-sdk-samples/Shared resources/Libraries/DemoTouchView/DemoTouchView.swift

```diff
@@ -358,7 +358,7 @@ private class TouchView: UIView {
         
         originalCenter = center
         
-        super.init(frame: CGRect(origin: CGPoint.zero, size:size))
+        super.init(frame: CGRect(origin: CGPoint.zero, size: size))
```

Comment:
Would be nice to drop the `CGPoint` here:

```suggestion
        super.init(frame: CGRect(origin: .zero, size: size))
```